t/react_template/README.md

```diff
@@ -0,0 +1,92 @@
+**This is not an official Google product.**
+
+# Template: React Framework
+
+This template was bootstrapped with [Create React App](https://github.com/facebook/create-react-app). 
+
+The react template enpower the front end development flow by adding automate Continuous Build (CB) and Continuous Testing (CT). GitHub Actions are leveraged to perform a series of pre-submit validations. 
+
+A branch protection rule will be established to enforce checks on each pull request including
+
+- Lint to provide static analysis (with ESlint)
+- Code format (with Prettier)
+- Build (with React-scripts)
+- Unit test and Component test
+- Integration test (with Cypress)
+
+
+## Set up instruction
+
+1. cd into an empty directory
+
+    ```shell
+    $ mkdir ~/react_template
+    $ cd ~/react_template
+    ```
+1. Install the `abc` binary
+
+    ```shell
+    $ go install github.com/abcxyz/abc/cmd/abc@latest
+    $ abc --help
+    ```
+
+    This only works if you have go installed (https://go.dev/doc/install) and have the Go binary directory in your $PATH (try PATH=$PATH:~/go/bin).
+
```

Comment:
How about we link to the top-level abc README instead of copying the installation instructions? We change the installation instructions pretty often. I know we used to do it this way, but I think we should stop.

Reply:
Done